zenoh/src/lib.rs

```diff
@@ -388,13 +393,15 @@ impl InitBuilder {
 
     /// Restrict the matching publications that will be receive by the [`Subscribers`](crate::subscriber::Subscriber)
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
     #[inline]
     pub fn subscribers_allowed_origin(mut self, origin: Locality) -> Self {
         self.subscribers_origin = origin;
         self
     }
     /// Restrict the matching queryables that will receive queries from this [`Session`]
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
Locality should not be configurable at session level but only per declare_subscriber and declare_queryable-basis.